pyproj/_crs.pyx

```diff
@@ -215,97 +473,439 @@ cdef class Ellipsoid:
+            full_name=self.full_name,
+            package_name=self.package_name,
+            url=self.unit_name,
+            direct_download=self.direct_download,
+            open_license=self.open_license,
+            available=self.available
+        )
+
+
+cdef class CoordinateOperation(Base):
+    """
+    Coordinate operation for CRS.
+
+    Attributes
+    ----------
+    name: str
+        The name of the method(projection) with authority information.
+    method_name: str
+        The method (projection) name.
+    method_auth_name: str
+        The method authority name.
+    method_code: str
+        The method code.
+    is_instantiable: int
+        If 1, a coordinate operation can be instantiated as a PROJ pipeline.
+        This also checks that referenced grids are available. 
+    has_ballpark_transformation: int
+        If 1, the coordinate operation has a “ballpark” transformation, 
+        that is a very approximate one, due to lack of more accurate transformations. 
+    accuracy: float
+        The accuracy (in metre) of a coordinate operation. 
+
+    """
+    def __cinit__(self):
+        self._params = None
+        self._grids = None
+        self.method_name = "undefined"
+        self.method_auth_name = "undefined"
+        self.method_code = "undefined"
+        self.is_instantiable = False
+        self.has_ballpark_transformation = False
+        self.accuracy = float("nan")
+
+    @staticmethod
+    cdef create(PJ_CONTEXT* projcontext, PJ* projobj):
+        cdef CoordinateOperation coord_operation = CoordinateOperation()
+        coord_operation.projobj = proj_crs_get_coordoperation(
+            projcontext,
+            projobj
+        )
+        if coord_operation.projobj == NULL:
+            return None
+
+        cdef char *out_method_name = NULL
+        cdef char *out_method_auth_name = NULL
+        cdef char *out_method_code = NULL
+        proj_coordoperation_get_method_info(
+            projcontext,
+            coord_operation.projobj,
+            &out_method_name,
+            &out_method_auth_name,
+            &out_method_code
+        )
+        coord_operation.method_name = decode_or_undefined(out_method_name)
+        coord_operation.method_auth_name = decode_or_undefined(out_method_auth_name)
+        coord_operation.method_code = decode_or_undefined(out_method_code)
+        coord_operation.name = "{method_name} [{method_auth_name}:{method_code}]".format(
+            method_name=coord_operation.method_name,
+            method_auth_name=coord_operation.method_auth_name,
+            method_code=coord_operation.method_code,
+        )
+        coord_operation.accuracy = proj_coordoperation_get_accuracy(
+            projcontext,
+            coord_operation.projobj
+        )
+        coord_operation.is_instantiable = proj_coordoperation_is_instantiable(
+            projcontext,
+            coord_operation.projobj
+        )
+        coord_operation.has_ballpark_transformation = \
```

Comment:
Maybe we should cast this (and the above as well) to booleans? Because now it are 0/1 ints

Reply:
Sounds like a better user-interfacing type.

Reply:
Updated, thanks!